content/en/faq/metrics-and-logs/metric-expiry.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How to work around missing metric expiry with in-proxy telemetry?
+weight: 20
+---
+
+First step is to identify the metric name and label which has high cardinality.
+Prometheus provides cardinality information at its `/status` page.
+Based on the labels which have high cardinality, there are several ways to reduce the cardinality:
+
+* `destination_service` could cause high cardinality since for intra mesh traffic, destination service could fallback to host header if Istio proxy does not know which service the request heads to.
```

Comment:
```suggestion
* Disable host header fallback. The `destination_service` label is one potential source of high-cardinality. The values for `destination_service` default to the host header if the Istio proxy is not able to determine which the destination service from other request metadata. If clients are using a variety of host headers, this could result in a large number of values for the  `destination_service`.
```